core/modules/layout/plugins/renderers/layout_renderer_standard.inc

```diff
@@ -534,7 +534,7 @@ class LayoutRendererStandard {
       $output = theme($style['region theme'], array('layout' => $this->layout, 'blocks' => $blocks, 'settings' => $style_settings, 'style' => $style));
     }
     else {
-      $output = implode('', $blocks);
+      $output = implode('', $blocks); // Use theme('layout_theme'...)?
```

Comment:
Let's just leave as-is with basic imploding when region theme is missing. If we make new theme functions/templates that aren't ever called it could cause additional confusion.